docs/pages/eas-update/getting-started.mdx

```diff
@@ -8,24 +8,16 @@ import { Step } from '~/ui/components/Step';
 
 Setting up EAS Update allows you to push critical bug fixes and improvements that your users need right away.
 
-## Prerequisites
```

Comment:
@jonsamp after discussion with @brentvatne , this prerequisites section should be removed, since all supported SDK versions (46 and greater) now support EAS Update.

Reply:
Thanks for updating this!